src/tokenGenerator.js

```diff
@@ -76,22 +78,29 @@ function updateTKK() {
       else {
         let res = await got('https://translate.google.com');
 
-        const code = res.body.match(/TKK=(.*?)\(\)\)'\);/g);
+        const code = res.body.match(/tkk:'\d+.\d+'/g);
+
+        if (code.length > 0) {
+          const xt = code[0].split(':')[1].replace('\'', '').replace('\'', '');
 
-        if (code) {
-          eval(code[0]);
-          /* eslint-disable no-undef */
-          if (typeof TKK !== 'undefined') {
-            window.TKK = TKK;
-            config.set('TKK', TKK);
-          }
-          /* eslint-enable no-undef */
+          window.TKK = xt;
+          config.set('TKK', xt);
         }
 
+        // if (code) {
+        //   eval(code[0]);
+        //   /* eslint-disable no-undef */
+        //   if (typeof TKK !== 'undefined') {
+        //     window.TKK = TKK;
+        //     config.set('TKK', TKK);
+        //   }
+        //   /* eslint-enable no-undef */
+        // }
```

Comment:
there is no point in keeping commented old code. either remove it or as much as possible from ol approach